mktrayicon.c

```diff
@@ -210,14 +211,16 @@ static GtkStatusIcon *create_tray_icon(char *start_icon)
 int main(int argc, char **argv)
 {
 	char *start_icon = "none";
+	char *tooltip = NULL;
+	char *pipe = NULL;
 	FILE *fifo;
 	GThread *reader;
 
 	XInitThreads(); /* see http://stackoverflow.com/a/18690540/472927 */
 	gtk_init(&argc, &argv);
 
 	if (argc == 1) {
-		printf("Usage: %s [-i ICON] [-t TOOLTIP] [FIFO]\n", *argv);
+		printf("Usage: %s [-i ICON] [-t TOOLTIP] [-p FIFO]\n", *argv);
```

Comment:
I'd prefer to keep `FIFO` as a positional argument. I believe `getopt` still supports that.